### PR TITLE
refactor(curriculum): colored markers step 16

### LINKED ISSUE #46899: Colored Markers - Step 16 - Awkward example placement
### Describe the Issue

Seen this issue come up a few times in the forum. At the bottom of the description is an example of setting multiple classes on an element, but it's just sort of placed there without any reference to it in the description. I've seen several people copy/paste from it, or something very similar to it (using a class name like "markerClass1" or "class1"). Since we aren't explaining the purpose of the example I think some people automatically assume that they need to use the example code to solve the step. Also, it seems a little strange that most of this step discusses color models and then right at the very end there is an example for something completely different. I think an explanation of what the example is for will make this transition a bit nicer.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-css-colors-by-building-a-set-of-colored-markers/step-16

### Your code

N/A

### Expected behavior

I don't have a great recommendation for fixing this yet because I'm not sure why the example is there. Is this the first time we have introduced people to multiple classes on an element? If so then it seems like we should probably explain in some detail how that works in words above the example. If it isn't, then is this just a friendly reminder? Or perhaps we can get rid of the example if we feel the user should know how to do this by now.

Also, I would tie the instructions into the color model a little better so people understand why they are adding these extra classes. Perhaps something like:

"To make each marker a different color you will need to add a unique class to each marker. First, add the additional class name `one` to the first marker `div` element."

P.S. If you wanted to shorten the description to make room for an explanation of the example then we could probably get rid of the first paragraph.

### Screenshots

_No response_

### System

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by bbsmooth**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46899

This step introduces the user to multiple classes on an element. Tried to incorporate the example into the instructions better with an explanation of how to add multiple classes.

### CODE REVIEW

**Review by hanswang123456 — COMMENTED** [review]
Looks good

**Review by jeremylt — DISMISSED** [review]

**Review by moT01 — COMMENTED** [review]
It feels like this step has/should have a new focus - teaching how to add multiple classes on an element. But it really introduces two new things now, that and the color models. I'm wondering if we should remove that first paragraph and move it somewhere else - maybe incorporate it into step 21. I think I would be in favor of something like that. What do you think @bbsmooth @jeremylt?

**Comment by bbsmooth** [comment]
@moT01 I'm on board with moving the first paragraph to a different step. It does seem like that information should be introduced when the user actually uses it, and step 21 seems like the first step that they do.

**Comment by Sembauke** [comment]
Hey Bruce,
Do you mind changing this in this pull-request. 

It seems like a good idea.

**Comment by bbsmooth** [comment]
@Sembauke Ya, don't know why I didn't do that already. I guess I didn't put two and two together correctly :smile:

**Review by jeremylt — DISMISSED** [review]
much better

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/617b65579ce424bf5f02ca73.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ dashedName: step-21
 
 # --description--
 
-Earlier you learned that the RGB color model is additive. This means that colors begin as black, and change as different levels of red, green, and blue are introduced.
+There are two main color models: the <dfn>additive</dfn> RGB (red, green, blue) model used in electronic devices, and the <dfn>subtractive</dfn> CMYK (cyan, magenta, yellow, black) model used in print. In this project you'll work with the RGB model.
```
Comment (moT01) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/617b65579ce424bf5f02ca73.md:10]:
```suggestion
There are two main color models: the <dfn>additive</dfn> RGB (red, green, blue) model used in electronic devices, and the <dfn>subtractive</dfn> CMYK (cyan, magenta, yellow, black) model used in print.
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/617b65579ce424bf5f02ca73.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ dashedName: step-21
...
 
-An easy way to see this is with the CSS `rgb` function.
+The RGB color model is additive. This means that colors begin as black, and change as different levels of red, green, and blue are introduced. An easy way to see this is with the CSS `rgb` function.
```
Comment (moT01) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/617b65579ce424bf5f02ca73.md:12]:
```suggestion
In this project, you'll work with the RGB model. This means that colors begin as black, and change as different levels of red, green, and blue are introduced. An easy way to see this is with the CSS `rgb` function.
```

**Review by moT01 — COMMENTED** [review]
Not sure if this is any better, but it sounded slightly redundant to me - so I tried to condense it a little

**Review by moT01 — APPROVED** [review]
LGTM @bbsmooth 🎉

**Review by Sembauke — APPROVED** [review]

**Pull request merged**